src/qt/walletcontroller.cpp

```diff
@@ -251,7 +252,7 @@ void CreateWalletActivity::askPassphrase()
 
 void CreateWalletActivity::createWallet()
 {
-    showProgressDialog(tr("Creating Wallet <b>%1</b>…").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
+    showProgressDialog(tr("Open Wallet"), tr("Creating Wallet <b>%1</b>…").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
```

Comment:
`Create Wallet` is more appropriate than `Open Wallet` here, no?
```suggestion
    showProgressDialog(tr("Create Wallet"), tr("Creating Wallet <b>%1</b>…").arg(m_create_wallet_dialog->walletName().toHtmlEscaped()));
```

Reply:
Addressed in  d9e421c. Thanks for pointing that out.